apps/users/url.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+from django.urls.conf import include
+from apps.users.views import *
+
+urlpatterns = [
+    path("djoser/", include("djoser.urls")),
+    path("token/", include("djoser.urls.jwt")),
+    path('role/', RoleViewSet.as_view(actions={'get': 'list', 'post': 'create'}), name='role'),
```

Comment:
do we need to define actions={'get': 'list', 'post': 'create'}, because I have not seen this for other modules, can you double check. Lets have the code unique